lib/wor/paginate/rspec.rb

```diff
@@ -28,20 +28,26 @@ def paginated_content
 
 RSpec::Matchers.define :be_paginated do
   match do |actual_response|
+    response = parse_response(actual_response)
     formatter = @custom_formatter || Wor::Paginate::Formatter
     @formatted_keys = formatter.new(MockedAdapter.new).format.as_json.keys
-    actual_response.keys == @formatted_keys
+    response.keys == @formatted_keys
+  end
+
+  def parse_response(response)
+    response.is_a?(Hash) ? response : JSON.parse(response.body)
```

Comment:
:thinking: What happens if response.boby be empty?
```rb
[1] pry(main)> JSON.parse([])
# => TypeError: no implicit conversion of Array into String
```

Reply:
response.body is a string. This method only supports json response.

Reply:
response.body could be binary data, if the request is of media_type `multipart/form-data`, but I think this is not those type of requests.